services/user.js

```diff
@@ -323,23 +357,9 @@ class User {
         if (!user || !Array.isArray(user) || user.length < 1 || user[0]?._id !== params.userID) {
             throw new Error(ERROR.USER_NOT_FOUND);
         }
-
         const updatedUser = { updateAt: sessionCurrentTime };
         const isCurator = updatedUser?.role === USER.ROLES.CURATOR || user[0]?.role === USER.ROLES.CURATOR || params?.role === USER.ROLES.CURATOR;
-        if (typeof(params.organization) !== "undefined" && params.organization && params.organization !== user[0]?.organization?.orgID) {
-            const result = await this.organizationCollection.aggregate([{
-                "$match": { _id: params.organization }
-            }, {"$limit": 1}]);
-            const newOrg = result?.[0];
-
-            if (!newOrg?._id || newOrg?._id !== params.organization) {
-                throw new Error(ERROR.INVALID_ORG_ID);
-            }
-
-            updatedUser.organization = orgToUserOrg(newOrg);
-        } else if ((typeof(params.organization) !== "undefined" && !params.organization && user[0]?.organization?.orgID)) { // Data Curator should not be assigned any Org
```

Comment:
Is this required for the current implementation? **The Data Curator should not be assigned any organization**. It was one of requirements in the past.